packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -17,4 +17,20 @@ describe("<DateRangeInput>", () => {
         const component = mount(<DateRangeInput />);
         expect(component.find(InputGroup).length).to.equal(2);
     });
+
+    it("startInputProps.inputRef receives reference to HTML input element", () => {
+        const inputRef = sinon.spy();
+        // full DOM rendering here so the ref handler is invoked
+        mount(<DateRangeInput startInputProps={{ inputRef }} />);
+        expect(inputRef.calledOnce).to.be.true;
+        expect(inputRef.firstCall.args[0]).to.be.an.instanceOf(HTMLInputElement);
```

Comment:
My experience is that in aggregate, small checks like this are valuable to ensuring API consistency in the event of future code refactors. I'd rather have a unit test tell me that I broke something than expend mental energy in a CR to make sure I/others didn't delete something small but important that used to be there. Other components already check for this `inputRef` stuff; I'd really like to do so here too.